R/fotmob_players.R

```diff
@@ -87,7 +88,7 @@ fotmob_get_match_players <- function(match_ids) {
         rn <- rownames(xt)
         suppressWarnings(
           xt2 <- matrix(
-            as.numeric(gsub("%", "", xt)),
+            as.character(xt),
```

Comment:
matrix values must all have the same type. so we need to be opinionated here

Reply:
Totally agree